roles/demo/vars/main.yml

```diff
@@ -1,3 +1,6 @@
 #keystone_adminurl: "{{ keystone_auth_protocol }}://{{ keystone_vip }}:{{ keystone_auth_port }}/v2.0"
 os_admin_auth: "--os-auth-url {{ keystone_adminurl}} --os-tenant-name {{admin_tenant}} --os-username {{admin_user }}  --os-password {{admin_pass}}"
 os_demo_auth: "--os-auth-url {{ keystone_adminurl}} --os-tenant-name {{demo_tenant}} --os-username {{demo_user }}  --os-password {{admin_pass}}"
+ext_gateway: "{{demo_ext_network | ipaddr('net') | ipaddr('1') | ipaddr('address')}}"
+start_ext_allocation: "{{demo_ext_network | ipaddr('net') | ipaddr('100') | ipaddr('address')}}" 
+end_ext_allocation: "{{demo_ext_network | ipaddr('net') | ipaddr('200') | ipaddr('address')}}" 
```

Comment:
Do we want to hard code the range start and end here?  Or is this a standard 'OpenStack' thing?